src/cpu/mod.rs

```diff
@@ -20,6 +20,9 @@ enum Instruction {
     SubImm(),
     SubHL(),
     // TODO: SbcHL
+    LdRegister(ArithmeticOperand, ArithmeticOperand),
+    LdRegisterHL(ArithmeticOperand),
```

Comment:
These names are confusing.  Maybe something like LdFromMem and LdToMem?  Will we need a new enum to handle register pairs since BC, DE, and HL are used to store memory addresses.  I'd like to avoid another enum.

Reply:
Maybe change `LdRegisterHL` to `LdFromMem` and take two `ArithmeticOperands`, where one is the register which contains the high byte and the other contains the low byte?

So it would look like:
```
Instruction::LdFromMem(ArithmeticOperand::A, ArithmeticOperand::H, ArithmeticOperand::L)
Instruction::LdFromMem(ArithmeticOperand::A, ArithmeticOperand::B, ArithmeticOperand::C)
Instruction::LdFromMem(ArithmeticOperand::A, ArithmeticOperand::D, ArithmeticOperand::E)
```